app/views/podcasts/_form.html.erb

```diff
@@ -19,7 +19,7 @@
     <div id="podcast-form-data" class="crayons-card mb-6 grid grid-flow-row gap-6 p-6">
       <div class="crayons-field crayons-field--checkbox">
         <%= check_box_tag :i_am_owner, class: "crayons-checkbox" %>
-        <%= label_tag :i_am_owner, "I am the owner of this podcast", class: "i_am_owner", class: "crayons-field__label" %>
+        <%= label_tag :i_am_owner, "I am the owner of this podcast", class: "crayons-field__label" %>
```

Comment:
removed the double class

Reply:
Out of curiosity, prior to your fix @rhymes, does the last `class` property overwrite the previous one or do they get concatenated?

Reply:
@nickytonline it gets overwritten, `crayons-field__label` is the only one that survives, see in the HTML of https://dev.to/podcasts/new